hilti/toolchain/src/compiler/driver.cc

```diff
@@ -1151,12 +1151,12 @@ Result<Nothing> Driver::initRuntime() {
     } catch ( const hilti::rt::Exception& e ) {
         printHiltiException(e);
         hookFinishRuntime();
-        rt::done();
+        finishRuntime();
```

Comment:
I don't quite understand this change: I don't see a `finishRuntime` function anywhere. Also, we have other places that call `rt::done`, would they need to change?

Reply:
See below.